### ISSUE TITLE
Some hidden projects can't be loaded on Dev

### ISSUE THREAD

**Issue by vic-topcoder** [appirio-tech/connect-app#2160]
### Expected behavior
Hidden project creation forms are not loading on DEV:
- https://connect.topcoder.com/new-project/enterprise-web
- https://connect.topcoder.com/new-project/enterprise-mobile
- https://connect.topcoder.com/new-project/topgear-dev
- https://connect.topcoder.com/new-project/caas-intake
- https://connect.topcoder.com/new-project/computer-vision
- https://connect.topcoder.com/new-project/data-exploration

Let's make sure the appropriate forms are loaded into DEV catalog and can be displayed correctly, and also test that all of the other `hidden` project forms are also available and loading correctly. It is important to transfer the catalog completely, as a lot of our partners do use the deep links to create projects.

@rmjmccormick FYI

**Comment by vic-topcoder** [comment]
The production catalog is at https://github.com/appirio-tech/connect-app/blob/master/src/config/projectWizard/index.js - that's where I referenced all the aliases to test, but it is time consuming and I didn't go through all the lines. Let's make sure we test all the URLs for the projects please.

**Comment by vikasrohit** [comment]
Yes, @vic-topcoder we are aware that production catalog and that is what we are trying replicate. As you it is time consuming, we tried to put all major non hidden and non disabled items first in dev. It is only about data entry which we made for production for June 28 release. There are still few project templates remaining on production, but again most of them are disabled. I don't think we need disabled form urgently because they are not accessible even on production right now.
For dev, we are trying to put missing templates, but as we are still in very early stages of database backed templates for projects and products.
